### PR TITLE
Pybind11

### PR DESCRIPTION
**Pull request by chGoodchild**

### CODE REVIEW

File: src/python/main.cpp [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #include <pybind11/iostream.h>
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
+#include <pybind11/complex.h>
```
Comment (wolfv) [inline src/python/main.cpp:4]: I don't think we need this one

File: src/python/main.cpp [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+#include <pybind11/functional.h>
+#include <pybind11/chrono.h>
```
Comment (wolfv) [inline src/python/main.cpp:6]: this one neither I think

Comment (chGoodchild) [reply src/python/main.cpp:6]: I get this when I leave it out:

Did you forget to `#include <pybind11/stl.h>`? Or <pybind11/complex.h>,
<pybind11/functional.h>, <pybind11/chrono.h>, etc. Some automatic
conversions are optional and require extra headers to be included
when compiling your pybind11 module.

Comment (chGoodchild) [reply src/python/main.cpp:6]: Perhaps there is a nicer workaround?

Comment (wolfv) [reply src/python/main.cpp:5]: yes, only remove chrono and complex.h

functional is necessary and used.

Comment (chGoodchild) [reply src/python/main.cpp]: makes sense...

**Pull request merged**